assets/css/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-weight: 300;
+    font-size: 1.05rem;
+    line-height: 1.6;
+    font-family: 'Poppins', sans-serif;
+}
+
+header {
+    background-color: #5C5B5B;
+    padding: 0px 15px;
+}
+
+header img { 
+    height: 200px;
+    width: 200px;
+}
+
+header h1 {
+    flex: auto;
+    font-size: 500%;
+    line-height: 25%;
+    color: #ffffff;
+    margin-bottom: 20%;
```

Comment:
This margin-bottom 20% is creating a big white space between the header and the h2.  I think it looks better if you just remove it.